docs/how-to/manage-external-contributors.md

```diff
@@ -0,0 +1,107 @@
+
+### Branch protection rules
+
+Create the following branch protection rules using the instructions [here](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#creating-a-branch-protection-rule):
+
+- Branch name pattern `**` with `Require signed commits` enabled.
+- Branch name pattern matching only the default branch of the repository, such as `main`, with the following enabled:
+  - `Dismiss stale pull request approvals when new commits are pushed`
+  - `Required signed commits`  
+  - `Do not allow bypassing the above settings`
+
+### Working with external contributors
+
+When `allow-external-contributor` is set to `false`, external contributors can still contribute through the following workflow:
+
+1. External contributors create pull requests as usual
+2. A repository maintainer with COLLABORATOR, MEMBER, or OWNER status reviews the code
+3. If the code is safe, the maintainer can:
+  - Approve and merge the pull request to another branch (workflows will run with the maintainer's permissions)
+  - Manually trigger workflow runs if needed (using workflow dispatch on the target branch)
+
+This approach ensures that all code from external contributors is reviewed by trusted users before execution on self-hosted runners.
+
+## Troubleshooting
+
+This section covers troubleshooting for common issues that users may encounter.
+
+### Workflows not running for external contributors
+
+If workflows are not running for external contributors when `allow-external-contributor=false`:
+
+1. Check the runner logs for authorization errors
+2. Verify the user's author association in the GitHub repository
+
+### Workflows running for untrusted users
+
+If you notice workflows running for users who shouldn't have access:
+
+1. Set `allow-external-contributor=false`
+2. Review repository collaborator permissions
+3. Check for any bypass rules in branch protection settings
+4. Audit recent workflow runs in the GitHub Actions tab
+
+## Migration from repo-policy-compliance
+
+If you were previously using the `repo-policy-compliance` charm, the `allow-external-contributor`
+configuration provides similar security controls with a simpler configuration model. The key
+differences are:
+
+- No separate charm deployment required
+- Configuration is handled directly through the github-runner charm
+- Broader event coverage (includes PR reviews and comments)
+- Simplified allow/deny model instead of complex policy rules
+
+To migrate:
+
+1. Remove the `repo-policy-compliance` charm integration
+2. Configure `allow-external-contributor=false` on the github-runner charm  
+3. Verify that external contributor workflows are properly restricted
```

Comment:
I believe there are not a lot of people using repo policy compliance out there. Should this not rather be part of the announcement we will make (e.g. in a discourse post), instead of living in the code documentation?